artifacts/kindClusterConfig/member2.yaml

```diff
@@ -2,6 +2,7 @@
 kind: Cluster
 apiVersion: "kind.x-k8s.io/v1alpha4"
 networking:
+  apiServerAddress: "{{host_ipaddress}}"
```

Comment:
@shauvet Can you explain why we need this?

Reply:
First, when I did local development, without it, I couldn't start my local api server, second, I thought it should be similar with the member1's config. Do I understand wrong?

Reply:
Thanks. @shauvet Could you please sign the commit as per https://github.com/karmada-io/dashboard/pull/87/checks?check_run_id=28865035951?

Reply:
> Thanks. @shauvet Could you please sign the commit as per https://github.com/karmada-io/dashboard/pull/87/checks?check_run_id=28865035951?

OK, done.